source/jobs/IotJobsClientWrapper.h

```diff
@@ -0,0 +1,119 @@
+#ifndef AWS_IOT_DEVICE_CLIENT_IOTJOBSCLIENTWRAPPER_H
```

Comment:
Don't forget to add the copyright and license statements

Reply:
Will do, thanks!